### ISSUE TITLE
No IPMI access to some UCL hosts

### ISSUE THREAD

**Issue by zerebubuth** [openstreetmap/operations#152]
From a first inspection it looks like the following hosts have no OOB access:

* eustace
* grindtooth
* noquiklos
* ridley
* urmel
* zark (because it's switched off?)

@tomhughes does that match the list you were expecting?

All those machines (except zark) are HP DL360 G6s, but so is clifford and it seems to be working. So it looks like we need to check the network config on those machines, then check with UCL's network team to see if the switch config is the same.

**Comment by tomhughes** [comment]
Yes I agree with that list. As far as I can see there are three possible causes:

* Something in the UCL network configuration.
* Something in the BIOS that controls which port is used when ILO is in "shared" mode - some HP boxes do allow you to choose but the ones I have dealt with let you change it with hponcfg but that doesn't seem to work here. There might be some hidden BIOS option though.
* The "shared" port is hardwired but we're using the other one. We seem to be using the same port on clifford as on other machines though.

**Comment by Firefishy** [comment]
Best solution is to get the default (untagged) VLAN switched to the internal VLAN. The untagged VLAN is meant to be 2801, but according to lldpctl it is currently 1001.

**Comment by tomhughes** [comment]
Don't they want us to release some ports though? Which means using shared OOB really...

**Comment by Firefishy** [comment]
Yes. shared OOB is fine and simpler to configure if the untagged VLAN gets set to 2801, we'd only then explicitly need to set VLAN 2800 when we required external access.

**Comment by tomhughes** [comment]
Well sure but every indication is that the shared mode is not working on those machines - if it was they should be OK as they support explicit VLAN tagging in shared mode and are set to 2801 already.
